fuzzing/fuzzer_worker.go

```diff
@@ -278,6 +287,11 @@ func (fw *FuzzerWorker) testNextCallSequence() (calls.CallSequence, []ShrinkCall
 			return true, err
 		}
 
+		// Add event values to copied ValueSet
+		lastExecutedSequenceElement := currentlyExecutedSequence[len(currentlyExecutedSequence)-1]
+		messageResults := lastExecutedSequenceElement.ChainReference.MessageResults()
+		valuegenerationtracer.AddTransactionOutputValuesToValueSet(messageResults, fw.valueSet)
```

Comment:
will need to update this line once the addition of values becomes the responsibility of the `ValueSet`